src/makeCalculator.js

```diff
@@ -38,6 +38,47 @@
  */
 function makeCalculator() {
   // write code here
+  const calculator = {
+    result: 0,
+
+    add(value) {
+      calculator.result += value;
+
+      return calculator;
+    },
```

Comment:
This method should just calculate and return result of calculation
Let's assign value to result in operate method